TMS.Net07.Homework.Task3.Part2/RecFibonacci.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TMS.Net07.Homework.Task3.Part2
+{
+    class RecFibonacci
+    {
+        public static void Fibonacci()
+        {
+            Console.WriteLine("How many numbers to deduce from the fibbonacci sequence: ");
+            int.TryParse(Console.ReadLine(), out var number);
+            for (var i = 0; i < number; i++)
+            {
+                Console.Write($"{Rec_Fibonacci(i)} ");
+            }
+            Console.ReadLine();
+        }
+        private static int Rec_Fibonacci(int number)
```

Comment:
In C# we usually use `PascalCase` for method naming
https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841